bigquery/storage/managedwriter/connection.go

```diff
@@ -71,6 +72,20 @@ func (pool *connectionPool) processWrite(pw *pendingWrite) error {
 	return conn.appendWithRetry(pw)
 }
 
+func (pool *connectionPool) addWriter(writer *ManagedStream) error {
+	if pool.router != nil {
+		return pool.router.writerAttach(writer)
+	}
+	return nil
```

Comment:
when there is no router available, shouldn't this return an error ?

Reply:
good point, updating